### ISSUE TITLE
crash when com.google.android.gms.vision.dynamite.face not found

### ISSUE THREAD

**Issue by mzealey** [onfido/onfido-android-sdk#46]
#### What was the expected behaviour?

Open up simple onfido on android sdk28 simulator WITHOUT google play installed (ie AOSP). Click on eg passport, then click back arrow to exit onfido. App crashes with the below output. Is there any way to include google play services within the app build to avoid this? Currently include the basic as you recommend in the main page. Looks like we are running 4.6.0 of sdk

```
02-21 21:30:19.454  3235  3422 W DynamiteModule: Local module descriptor class for com.google.android.gms.vision.dynamite.face not found.
02-21 21:30:19.454  3235  3421 W GooglePlayServicesUtil: Google Play Store is missing.
02-21 21:30:19.454  3235  3422 E ActivityThread: Failed to find provider info for com.google.android.gms.chimera
02-21 21:30:19.454  3235  3422 W DynamiteModule: Failed to retrieve remote module version.
02-21 21:30:19.454  3235  3422 W GooglePlayServicesUtil: Google Play Store is missing.
02-21 21:30:19.454  3235  3422 I DynamiteModule: Considering local module com.google.android.gms.vision.dynamite.face:0 and remote module com.google.android.gms.vision.dynamite.face:0
02-21 21:30:19.454  3235  3422 D FaceNativeHandle: Cannot load feature, fall back to load whole module.
02-21 21:30:19.455  3235  3422 W DynamiteModule: Local module descriptor class for com.google.android.gms.vision.dynamite not found.
02-21 21:30:19.455  3235  3422 W GooglePlayServicesUtil: Google Play Store is missing.
02-21 21:30:19.455  3235  3422 I DynamiteModule: Considering local module com.google.android.gms.vision.dynamite:0 and remote module com.google.android.gms.vision.dynamite:0
02-21 21:30:19.455  3235  3422 E FaceNativeHandle: Error Loading module
02-21 21:30:19.455  3235  3422 E FaceNativeHandle: com.google.android.gms.dynamite.DynamiteModule$LoadingException: No acceptable module found. Local version is 0 and remote version is 0.
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.dynamite.DynamiteModule.load(Unknown Source:168)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.internal.vision.zzl.zzp(Unknown Source:71)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.vision.face.internal.client.zza.<init>(Unknown Source:9)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.vision.face.FaceDetector$Builder.build(Unknown Source:33)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.internal.firebase_ml.zzib.zzey(Unknown Source:164)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.internal.firebase_ml.zzha.zzfb(Unknown Source:37)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.internal.firebase_ml.zzha.call(Unknown Source:0)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.internal.firebase_ml.zzgp.zza(Unknown Source:0)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at com.google.android.gms.internal.firebase_ml.zzgq.run(Unknown Source:4)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at android.os.Handler.handleCallback(Handler.java:873)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at android.os.Handler.dispatchMessage(Handler.java:99)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at android.os.Looper.loop(Looper.java:193)
02-21 21:30:19.455  3235  3422 E FaceNativeHandle:      at android.os.HandlerThread.run(HandlerThread.java:65)
02-21 21:30:19.455  3235  3422 W FaceNativeHandle: Native handle not yet available. Reverting to no-op handle.
```

**Comment by mzealey** [comment]
(just to note in the android emulator with google play installed it works fine)

**Comment by sergiomarquesmoura** [comment]
Hi @mzealey . Thanks for flagging this. That's true, at the moment we are not supporting devices without play services but we are investigating it. I will update you once I have further info

**Comment by sergiomarquesmoura** [comment]
Hi @mzealey . We just released our `4.7.0` version, which solves this issue. Full changelog [here](https://github.com/onfido/onfido-android-sdk/blob/master/CHANGELOG.MD#470).
